certified-connectors/Impexium/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#D7212F",
+    "capabilities": [],
+    "publisher": "Impexium Corporation"
```

Comment:
@RjImpx Please add `stackOwner` property as well.
Please refer [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#publisher-and-stack-owner)